Cargo.lock

```diff
@@ -6640,6 +6640,7 @@ dependencies = [
  "arrow-schema",
  "async-trait",
  "auto_enums",
+ "bincode 2.0.0-rc.3",
```

Comment:
`EpochPair` needs this to derive `Encode` and `Decode`. It needs these traits because `Operation::LocalStorageInit` now takes in `EpochPair` instead of `u64`.